python/main_loop.py

```diff
@@ -239,10 +239,12 @@ def createNewLocalPath(sailbot, maxAllowableRuntimeSeconds, desiredHeadingPublis
         localPath.updateWindDirection(state)
         localPath.updateObstacles(state)
 
-        # Publish nextLocalWaypoint and nextGlobalWaypoint and path cost
+        # Publish nextLocalWaypoint and nextGlobalWaypoint and path cost. Log nextGlobalWaypoint
         nextLocalWaypointPublisher.publish(localPath.getNextWaypoint())
         previousLocalWaypointPublisher.publish(localPath.getPreviousWaypoint())
         nextGlobalWaypointPublisher.publish(state.globalWaypoint)
+        globalWaypointLogStr = "Next Global Waypoint is: (lat: {}, lon: {})"
```

Comment:
The text looks great! All nice on one line. 

Can you put this into the loginfo directly? No need to keep the `globalWaypointLogStr` around

```
rospy.loginfo("Next Global Waypoint is: (lat: {}, lon: {})".format(state.globalWaypoint.lat, state.globalWaypoint.lon))
```